### ISSUE TITLE
Auto language setting

### ISSUE THREAD

**Issue by JulianKniephoff** [elan-ev/opencast-studio#1097]
@lkiesow wrote in https://github.com/elan-ev/opencast-studio/pull/1069#issuecomment-1691158109:

> ![Screenshot from 2023-08-24 09-08-52](https://github.com/elan-ev/opencast-studio/assets/1008395/f805e435-4ca4-4039-8da3-d5a91bc56618)
>
> “Appearence” has a mode “Auto” to select the system design. Language should also have an auto-option to select the preferred system language.

with a 👍 from @oas777.

**Comment by LukasKalbertodt** [comment]
Yeah I think that's exactly the plan: clear the local storage for i18next so that the auto detection kicks in again.

And yes, you are right, the top comment should mention that i18next already does an automatic language selection based on the browser. I am pretty sure, that browser basically always provide this information, I'm not even sure a normal user could configure popular browser in a way to break this. 

I personally assume that the auto language works perfectly for >99% of users. The remaining ones use the language menu to set one specific language. I doubt that any real user ever wanted to have this "auto" setting in the language menu. What for? Unlike a color scheme, which could conceivably change regularly (e.g. day/night with light/dark scheme), I don't see any situation where the browser language changes regularly. 
So in my mind this issue is very very very low priority. Almost to the point of closing it.
